webapp/components/navbar.jsx

```diff
@@ -786,18 +786,14 @@ export default class Navbar extends React.Component {
             );
 
             isChannelAdmin = ChannelStore.isChannelAdminForCurrentChannel();
+            channelTitle = channel.display_name;
```

Comment:
I don't think this will work if the current user's name is first in alphabetical order since the channel display name for group messages includes the current user's name but the name we display in the sidebar doesn't show the current user's name

Reply:
Oh just noticed your comment about redux. If that sets the display name before we get here then it probably just works

Reply:
Glad you saw my comment on that referenced PR and sorry if I failed to indicate here :)
Yeah, I think it's already processed at mattermost-redux. I hope it's correct.